### ISSUE TITLE
[Bug]: FTB Quests—hit target is not rotated along with rotated images

### ISSUE THREAD

**Issue by SubordinalBlue** [FTBTeam/FTB-Mods-Issues#1281]
### Mod

FTB Quests

### Mod version

Any NeoForge version, at least, afaik.

### Forge / Fabric version

NeoForge 1.21

### Modpack & version

Craftoria... all versions we've produced so far. xD

### What issue are you having?

Easier to explain with a visual reference, I think.

In the example quest chapter design below, the horizontal ae2 spatial pylon images can be difficult to work with, because their hit target for right clicking is not rotated to match their rotation (90° in this case). The user (quest editor) is left to guess-and-try-again to find the right place to right click to work with a rotated image (in this example the problem is exacerbated much by it also being a stretched texture).

<img width="501" alt="Screenshot 2024-07-18 at 7 01 06 PM" src="https://github.com/user-attachments/assets/ec7380c7-1a07-4062-b366-7a2ccd6f6cdb">

### Steps to reproduce

1. Place image
2. Rotate it (optionally stretch it out to make the resulting problem much more annoying)
3. Try to right-click on it... it will only work where the image _would_ be if it wasn't rotated.

### Anything else to note?

Nope.

**Comment by desht** [comment]
Yeah, it's a known issue, and one that's rather difficult to resolve, since the vanilla game doesn't support the concept of click areas which are rotated to anything other than the cardinal compass points.

I guess one option might be to show a hitbox where the image _can_ be clicked (some kind of highlighted rectangular area).

**Comment by desht** [comment]
Fixed in current releases

**Comment by SubordinalBlue** [comment]
Sweet; thank you.
